### DISCUSSION TITLE
Fetching local packages failed...

### DISCUSSION THREAD

**Discussion by zoiba-zia** [atom/atom#22664] [Q&A]
Hello, 
Atom Version: 1.57.0
Device Specifications: Windows 10 home 
Version: 20H2
x64-based processor

So, I downloaded atom directly from Github
but I am unable to download or even load pre-installed packages and themes. Is there any way to fix this? 

I recently started coding and would like help in beginner-friendly terms. Thank you 

![Screenshot 2021-06-27 132128](https://user-images.githubusercontent.com/73377489/123536919-a0303e00-d74a-11eb-8625-7dd502dc6402.png)

![Screenshot 2021-06-27 132055](https://user-images.githubusercontent.com/73377489/123536935-b1794a80-d74a-11eb-90e4-da5eab38171e.png)

![Screenshot 2021-06-27 132030](https://user-images.githubusercontent.com/73377489/123536939-b63dfe80-d74a-11eb-9103-0167db98b22d.png)

**Comment by lenora2468B** [comment]
hi- i had the hardest time with that application as well, or all applications. i am still trying to use my key.

**Reply by zoiba-zia** [reply]
How did you even get started with it? I chose to use Atom because so many senior developers recommended that it is easier to work with Git through it. I don't know anymore. 

I got this when I put it in my powershell 

![Screenshot 2021-06-27 202936](https://user-images.githubusercontent.com/73377489/123549505-02a72f80-d787-11eb-99e6-0c51a3c94686.png)

**Reply by icecream17** [reply]
@abioz-aiz
"path" is a bunch of places where the system looks for a file
Usually "path" is set to have Atom in it, but maybe because of the different installings, it isn't set this time

Do you have `AtomSetup-x64.exe` or something like that?


An alternative way to open atom is just by searching for it
![image](https://user-images.githubusercontent.com/58114641/123553351-475ba800-d740-11eb-8c9f-e1ddae862558.png)
